tests/test.single-page-no-attachment.js

```diff
@@ -24,12 +18,10 @@ fixture`Single page cartridge (compact)`
   "/test-cartridges/single-page/imsmanifest.xml"
 )}`;
 
-test("Header and Nav is hidden", async t => {
+test("Header is hidden", async t => {
   await t
     .expect(Selector("h1").withText(`Our Purpose`).exists)
     .ok()
-    .expect(Selector("nav").exists)
-    .notOk()
```

Comment:
ditto

Reply:
@cguanzon yeah, this one shouldn't be removed. there is still some work that needs to be done on tests so I'm gonna close this pr while I resolve those issues. thanks!